oio/directory/meta2.py

```diff
@@ -0,0 +1,319 @@
+# Copyright (C) 2018-2019 OpenIO SAS, as part of OpenIO SDS
```

Comment:
Why 2018?

Reply:
Part of the code in this file is already used in `oio/directory/meta1.py`.
And `oio/directory/meta2.py` replace `oio/directory/meta1.py`.